__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("./../db/app/app")
+const request = require("supertest")
+const {topicData, userData, commentData, articleData} = require("./../db/data/test-data/index")
+const db = require("./../db/connection")
+const seed = require("./../db/seeds/seed")
+
+beforeEach(() => seed({ topicData, userData, commentData, articleData }))
+afterAll(() => db.end())
+
+describe("GET /api/topics", () => {
+    test("responds with status code 200", () => {
+        return request(app).get('/api/topics').expect(200)
+    })
+    test("returns an array of all topic objects with required properties", () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            const topicObj = {
+                description: expect.any(String),
+                slug: expect.any(String)
+            }
+            expect(body.topics.length === 3)
```

Comment:
This doesn't check the length is 3 we need to use a jest Matcher expect(something).toBe(something) or expect(something).toHaveLength(something)